src/services/midgard.ts

```diff
@@ -47,4 +47,7 @@ export const reloadPools = () => reloadPools$$.next(0)
 /**
  * Pool data
  */
-export const pools$ = reloadPools$$.pipe(exhaustMap((_) => poolsRD$))
+export const pools$ = reloadPools$$.pipe(
+  exhaustMap((_) => poolsRD$),
+  shareReplay()
```

Comment:
^ This change is for caching data of `pools` (as mentioned in Slack today). It just came into this PR by accident. _Will leave it here..._